### ISSUE TITLE
Problem with margin

### ISSUE THREAD

**Issue by fahad5359** [ryanpcmcquen/obsidian-org-mode#13]
I am encountering an issue when runing the plug-in. Where the text goes waay to the right. after investigation i found that the problem with the margin.
```HTML
<div class="CodeMirror-sizer" style="margin-left: 1314px; margin-bottom: 0px; border-right-width: 50px; min-height: 527px; min-width: 848.763px; padding-right: 0px; padding-bottom: 0px;">
```
After updating the `margin-left: 1314px;` to `margin-left: 100px;` the issue is gone...

Photo attached
Regards

![Capture](https://github.com/ryanpcmcquen/obsidian-org-mode/assets/75564334/e008fc54-e38c-4456-9832-1c34a7814c73)

![Capture2](https://github.com/ryanpcmcquen/obsidian-org-mode/assets/75564334/9beba3c0-cda7-48d0-9367-713b5ba09a60)

**Comment by ryanpcmcquen** [comment]
What theme is that? Does it happen with the default theme?

**Comment by fahad5359** [comment]
Yep, this is the the default theme

**Comment by aareman** [comment]
Having the same issue, it just started.

**Comment by zzeitt** [comment]
Same issue here on mobile (iOS).

> Obsidian version: 1.15.12 (134)

![image](https://github.com/ryanpcmcquen/obsidian-org-mode/assets/30381410/32255f82-1a73-44d6-9b34-636c08f73848)

**Comment by ryanpcmcquen** [comment]
Very busy at the moment, but if someone has a PR to fix this I will merge/release it.

**Comment by nohzafk** [comment]
i can confirm that remove the gutter setting will solve the margin issue

`src/main.ts`

```diff
-            foldGutter: {
-                minFoldSize: 1,
-            },
-            foldOptions: {
-                widget: " ...",
-            },
-            gutters: ["CodeMirror-foldgutter"],
+           gutters: [],

```

**Comment by ryanpcmcquen** [comment]
@nohzafk did you try that with only changing the gutter settings (and keeping the folding setting)?

**Comment by nohzafk** [comment]
I tried, I can open the org file on both cases, but the format is not rendered correctly.

**Comment by ryanpcmcquen** [comment]
Fixed in 1.4.1, although this could still be improved.
